app/src/main/java/com/futurice/freesound/app/module/ApiModule.java

```diff
@@ -107,19 +111,18 @@ Gson provideGson() {
 
     @Provides
     @Singleton
-    RequestInterceptor provideRequestInterceptor(@Named(API_TOKEN_CONFIG) String apiToken) {
-        return new FreeSoundApiInterceptor(apiToken);
+    OkHttpClient provideApiOkHttpClient(@AppInterceptors List<Interceptor> appInterceptor,
```

Comment:
Now that the API token is in the header, there should be a specific instance of OkHttp for Retrofit responses, restricted by a qualifier annotation; otherwise the client could be unintentionally reused with an unwanted request header parameter.